aws/resource_aws_emr_cluster_test.go

```diff
@@ -98,6 +98,17 @@ func TestAccAWSEMRCluster_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "bootstrap_action.#", "0"),
 				),
 			},
+			{
+				Config: testAccAWSEmrClusterConfigMultipleSubnets(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSEmrClusterExists(resourceName, &cluster),
```

Comment:
For completeness, this step should include a call to `testAccCheckAWSEmrClusterRecreated()`, since both `subnet_id` and `subnet_ids` are `ForceNew`